vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +52,83 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormatUsingAdminCli(volName string, hostname string) string {
```

Comment:
I do not see why it is important ... E.g. I'd prefer to get all results in one 2 seconds call, rather than in 10 2 seconds calls.